pkg/clientaccess/token.go

```diff
@@ -272,11 +281,16 @@ func (i *Info) Get(path string) ([]byte, error) {
 	}
 	p.Scheme = u.Scheme
 	p.Host = u.Host
-	return get(p.String(), GetHTTPClient(i.CACerts, i.CertFile, i.KeyFile), i.Username, i.Password, i.Token())
+	return get(p.String(), GetHTTPClient(i.CACerts, i.CertFile, i.KeyFile, DefaultClientTimeout), i.Username, i.Password, i.Token())
 }
 
-// Put makes a request to a subpath of info's BaseURL
+// Put makes a request to a subpath of info's BaseURL, with a 10 second timeout
 func (i *Info) Put(path string, body []byte) error {
+	return i.PutWithTimeout(path, body, DefaultClientTimeout)
+}
+
+// PutWithTimeout makes a request to a subpath of info's BaseURL, but with a custom timeout
+func (i *Info) PutWithTimeout(path string, body []byte, timeout time.Duration) error {
```

Comment:
Rather than adding another method with a slightly different signature, might it be a little cleaner to change Put/Get and so on to take options to customize the behavior? We already do that for ValidationOptions in pkg/clientaccess. Maybe something like:

```golang
type ClientOption func(*http.Client)

func WithTimeout(d time.Duration) ClientOption {
  return func(c *http.Client) {
    c.Timeout = d
  }
}

func (i *Info) Put(path string, body []byte, option ...ClientOption) error {}

info.Put("/v1-"+version.Program+"/encrypt/config", b, clientaccess.WithTimeout(90*time.Second))
```